pkgs/by-name/pd/pdfarranger/package.nix

```diff
@@ -20,7 +21,18 @@ python3Packages.buildPythonApplication rec {
     hash = "sha256-94qziqJaKW8/L/6+U1yojxdG8BmeAStn+qbfGemTrVA=";
   };
 
-  nativeBuildInputs = [ wrapGAppsHook3 ];
+  nativeBuildInputs =
+    [ wrapGAppsHook3 ]
+    ++ lib.optionals pkgs.stdenv.isDarwin [
+      pkgs.gettext
+    ];
+
+  patchPhase =
+    ''''
+    + lib.optionals pkgs.stdenv.isDarwin ''
+      LINTL="${lib.getLib pkgs.gettext}/lib/libintl.8.dylib"
+      sed -i "s@return 'libintl.8.dylib'@return '$LINTL'@" pdfarranger/pdfarranger.py
```

Comment:
please use `substituteInPlace --replace-fail` or a patch instead. If the code you are patching changes, sed will silently do nothing and linux people will not notice the breakage.